src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/form_csrf_disabled.php

```diff
@@ -4,6 +4,7 @@
     'annotations' => false,
     'csrf_protection' => false,
     'form' => [
+        'enabled' => true,
```

Comment:
While we change tests to make this PR pass, this won't affect end user's project: the auto-enable will still rely on the required packages being installed. Here, we have to be explicit because FullStack::class is found.